src/components/Button/Button.module.css

```diff
@@ -0,0 +1,27 @@
+.button__wrap{
+    display: flex;
+    justify-content: center;
+    margin: 14px 0 14px 0;
+}
+
+.button {
+    width: 73px;
+    height: 32px;
+    border: 1px solid #999999;
+    box-sizing: border-box;
+    border-radius: 37px;
+    outline: none;
+    cursor: pointer;
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 14px;
+    line-height: 16px;
+    color: #356EFF;
+    margin-right: 8px;
+}
+
+.button:hover {
+    color: #ffffff;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tododo/pullRequest?prid=5504860)